src/route/nodes/netid/components/CreateEgress.tsx

```diff
@@ -16,7 +16,16 @@ import { useTranslation } from 'react-i18next'
 import { useDispatch } from 'react-redux'
 import { createEgressNode } from '~modules/node/actions'
 import { useNodeById } from '~util/node'
-import { NmForm, NmFormInputText } from '~components/form'
+import { NmForm, NmFormInputText, validate } from '~components/form'
+import { correctIPv4CidrRegex, correctIpv6Regex } from '~util/regex'
+
+const convertStringToArray = (commaSeparatedData: string) => {
```

Comment:
move this function to a util and replace everywhere its called with that reference